Client/Frontend/Settings/Brave News/BraveNewsDebugSettingsView.swift

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import SwiftUI
+import BraveShared
+
+extension FeedDataSource.Environment {
+  fileprivate var name: String {
+    switch self {
+    case .dev: return "Dev"
+    case .staging: return "Staging"
+    case .production: return "Production"
+    }
+  }
+}
+
+extension FeedDataSource {
+  fileprivate func description(of state: State) -> String {
+    switch state {
+    case .initial:
+      return "—"
+    case .loading:
+      return "Loading"
+    case .success:
+      return "Success"
+    case .failure:
+      return "Error"
+    }
+  }
+}
+
+struct BraveNewsDebugSettingsView: View {
+  private let feedDataSource: FeedDataSource
+  private let dismissAction: (() -> Void)?
+  
+  init(dataSource: FeedDataSource, dismissAction: (() -> Void)? = nil) {
+    self.feedDataSource = dataSource
+    self.dismissAction = dismissAction
+    // These values should always be overridden
+    self._environment = .init(wrappedValue: dataSource.environment)
+    self._languageCode = .init(wrappedValue: dataSource.languageCode)
+  }
+  
+  @State private var environment: FeedDataSource.Environment = .production
+  @State private var languageCode: String = "en"
+  
+  var body: some View {
+    List {
+      Section {
+        Picker("Environment", selection: $environment) {
+          ForEach(FeedDataSource.Environment.allCases, id: \.self) { env in
+            Text(env.name)
+          }
+        }
+        NavigationLink {
+          LanguagePicker(languageCode: $languageCode)
+        } label: {
+          HStack {
+            Text("Language")
```

Comment:
Do we not enforce `verbatim` texts anymore since we disabled it globally?

Reply:
Yeah no need since SwiftUI Text localization is disabled, it also breaks putting Images inside Text since it disables string interpolation

Reply:
this is good to know, for our swiftUI guidelines